warp/handlers/signature_request.go

```diff
@@ -54,7 +55,16 @@ func (s *signatureRequestHandler) OnSignatureRequest(ctx context.Context, nodeID
 	if err != nil {
 		log.Debug("Unknown warp signature requested", "messageID", signatureRequest.MessageID)
 		s.stats.IncSignatureMiss()
-		return nil, nil
+
+		// Return empty response because the message was not found in backend,
+		// instead of returning nil, which would drop message response.
+		emptyResponse := message.SignatureResponse{Signature: [bls.SignatureLen]byte{}}
+		emptyResponseBytes, err := s.codec.Marshal(message.Version, &emptyResponse)
```

Comment:
Do we handle this case on the receiver side?

Also can we pre-calculate `emptyResponse` and store it in a var?

Reply:
Currently the codec is part of the signature request handler, so this would require re-factoring to hardcode the codec we use.